config/prometheus/README.md

```diff
@@ -29,3 +29,20 @@ The files [install-prometheus.yaml](install-prometheus.yaml) and
 [servicemonitors.yaml](servicemonitors.yaml) are taken from
 [istio/tools](https://github.com/istio/tools/), with configuration related to
 Istio left out.
+
+### Prometheus scrape_interval
+
+`scrape_interval` is how frequently the Prometheus to scrape targets, by default
```

Comment:
The scrape interval (`scrape_interval`) determines how often Prometheus scrapes targets. The default value for this interval is [1m](...). We change this interval in our configuration to **1s**, since our tests typically last only 30s. See <...> for details.